### PR TITLE
Trash pile tweaks

### PR DESCRIPTION
**Pull request by SpadesNeil**
- Adds zip gun
- Removes vault moneybag (glitchy)
- Adds makeshift metal armor
- Removes press armor (confusing for SOP)

### CODE REVIEW

**Comment by JoanRisu** [comment]
The dart gun shoots chem darts like a Tranq Dart from MGS

File: code/game/objects/structures/trash_pile.dm [hunk 1]
```diff
@@ -21,9 +21,9 @@
 		/obj/item/device/perfect_tele,
 		/obj/item/weapon/bluespace_harpoon,
 		/obj/item/weapon/gun/energy/netgun,
-		/obj/item/weapon/moneybag/vault,
+		/obj/item/weapon/gun/projectile/pirate,
 		/obj/item/weapon/permit,
-		/obj/item/weapon/gun/projectile/dartgun
+		/obj/item/clothing/mask/gas/voice
```
Comment (Cyantime) [inline code/game/objects/structures/trash_pile.dm]: Voice changer is already on the list, just above the emag.

Comment (SpadesNeil) [reply code/game/objects/structures/trash_pile.dm]: oh derp, well what else should I put?

**Comment by SpadesNeil** [comment]
I'll put dart guns back, then. @JoanRisu

File: code/game/objects/structures/trash_pile.dm (hunk 1 above)
Comment (Cyantime) [reply code/game/objects/structures/trash_pile.dm]: Maybe throw in a /obj/item/device/batterer? It tends to be forgotten and is some sort of weird science thing.

**Comment by Ketrai** [comment]
wha? press armor confusing? doesn't it say what it is? armor for journalists, literally all it is. should be wearable in code green

**Comment by SpadesNeil** [comment]
>wha? press armor confusing? doesn't it say what it is? armor for journalists, literally all it is. should be wearable in code green

So Johnny McToolbox shouldn't be able to get it without stealing it from the station's Journalist (Librarian).

**Comment by Yoshax** [comment]
I don't believe it's mapped in for the Librarian.

**Comment by Renkindle** [comment]
Never seen it before when I've played as Librarian myself, so yeah, I don't think it's mapped.

**Comment by SpadesNeil** [comment]
I will add that in during a different PR.

**Pull request merged**